template.yml

```diff
@@ -96,6 +96,127 @@ Resources:
           /category/delete/{category_id}:
             delete: *standardImplementation
 
+  # The AWS S3 bucket into which our SPA build files will be deployed
+  S3Bucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: !Sub "react-cors-spa-${RestApi}"
```

Comment:
Why is the api here?

Reply:
to use as the name of the bucket

Reply:
I wouldn't. The bucket has little to do with the api. I would include the name of the project maybe but you don't need the api id.